.gitignore

```diff
@@ -28,4 +28,6 @@ tests/fixtures/project/.acquia-cli.yml
 
 cx-api-spec
 dns-records.yaml
-dns-records.json
\ No newline at end of file
+dns-records.json
```

Comment:
I'd recommend adding a comment to indicate `dns-records.json` and `subscription-*` is for Platform email specifically. Would prevent `git blame` a year from now when we don't recall why it was added.